source/Outfit.h

```diff
@@ -48,6 +48,7 @@ class Outfit : public Weapon {
 	bool IsDefined() const;
 
 	const std::string &Name() const;
+	const std::string &DisplayName() const;
```

Comment:
To stay consistent, use `Name` for the display name and for the actual name `TrueName`.

Reply:
done